### PR TITLE
feat(Amazon Kindle): add presence

### PR DESCRIPTION
**Pull request by PrincessAkira**
## Description 
Addition of Amazon Kindle as a service

## Acknowledgements
- [X] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [X] I linted the code by running `yarn format`
- [X] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

image

image

</details>

### CODE REVIEW

**Comment by PrincessAkira** [comment]
Thats funny.
I ran yarn format and it fails

File: websites/A/Amazon Kindle/metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "202740603790819328",
+		"name": "aimslut"
+	},
+	"service": "Amazon Kindle",
+	"description": {
+		"en": "Read books online via Amazon Kindle"
```
Comment (theusaf) [inline websites/A/Amazon Kindle/metadata.json:9]: Consider using one of the official descriptions instead
```suggestion
		"en": "Access and read millions of titles instantly and comfortably on your phone, tablet, or computer."
```

**Review by theusaf — CHANGES_REQUESTED** [review]

**Review by theusaf — APPROVED** [review]

File: websites/A/Amazon Kindle/presence.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+const presence = new Presence({
+		clientId: "1208798725116395592",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			details: "Stöbert in der Kindle Bibliothek",
```
Comment (Bas950) [inline websites/A/Amazon Kindle/presence.ts:8]: Don't think this should be in german...

Comment (PrincessAkira) [reply websites/A/Amazon Kindle/presence.ts]: You are correct.
This was in when i originally wanted to make it for the german version online

**Review by theusaf — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**